src/agent/reqwest-retry/src/lib.rs

```diff
@@ -173,17 +181,34 @@ impl SendRetry for reqwest::RequestBuilder {
 mod test {
     use super::*;
 
-    // TODO: convert to feature-gated integration test.
-    #[ignore]
     #[tokio::test]
-    async fn empty_stack() -> Result<()> {
-        let resp = reqwest::Client::new()
-            .get("http://localhost:5000/api/testGet")
+    async fn retry_should_pass() -> Result<()> {
+        reqwest::Client::new()
+            .get("https://www.microsoft.com")
```

Comment:
```suggestion
            .get("https://www.contoso.com")
```

Reply:
Disagree with this.  While the URL exists, it redirects to www.microsoft.com.  This makes it clear in the test that it's making a request to a microsoft property, where contoso is not.